### PR TITLE
doc/rados/configuration: add more scrub related config

### PR DESCRIPTION
**Pull request by tchaikov**
Signed-off-by: Kefu Chai <kchai@redhat.com>

### CODE REVIEW

File: doc/rados/configuration/osd-config-ref.rst [hunk 1]
```diff
@@ -219,6 +219,23 @@ scrubbing operations.
...
+:Type: Integer in the range of 0 to 24
+:Default: ``0``
+
+
+``osd scrub end hour``
+
+:Description: The time of day for the upper bound when a scheduled scrub can be
+              performed. Along with ``osd scrub begin hour``, they define a time
+              window, in which the scrubs can happen. But a scrub will be performed
+              no matter the time window allows not not, as long as the placement
```
Comment (dachary) [inline doc/rados/configuration/osd-config-ref.rst]: s/not not/not/

**Comment by dachary** [comment]
@tchaikov a typo, otherwise :+1: :-)

<pre>
Reviewed-by: Loic Dachary &lt;ldachary@redhat.com>
</pre>

File: doc/rados/configuration/osd-config-ref.rst (hunk 1 above)
Comment (tchaikov) [inline doc/rados/configuration/osd-config-ref.rst]: @dachary  how about s/not not/or not/ ?

Comment (dachary) [inline doc/rados/configuration/osd-config-ref.rst]: right thanks for fixing the typo of my typo fix ! :-)

**Comment by tchaikov** [comment]
@dachary thanks for your review!

**Pull request merged**